src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1048,6 +1048,32 @@
   (jdbc/do-commands
     "CREATE INDEX idx_certnames_latest_report_id on certnames(latest_report_id)"))
 
+(defn add-producer-to-reports-catalogs-and-factsets
+  []
+  (jdbc/do-commands
+    "CREATE TABLE producers (
+        id bigint PRIMARY KEY,
+        name text NOT NULL UNIQUE)"
+    "CREATE SEQUENCE producers_id_seq
+        START WITH 1
+        INCREMENT BY 1
+        NO MINVALUE
+        NO MAXVALUE
+        CACHE 1"
```

Comment:
yeah, @senior just was suggesting that I follow what was already in there for certain things.
Like, can't the whole CREATE SEQUENCE thing be taken care of by adding 'serial' when creating the id column?

I will make your edits though, if that's how we generally write things.